frontend/packages/operator-lifecycle-manager/src/components/operand/DEPRECATED_operand-form.tsx

```diff
@@ -779,9 +779,9 @@ export const DEPRECATED_CreateOperandForm: React.FC<OperandFormProps> = ({
           <dt>Limits</dt>
           <dd>
             <ResourceRequirements
-              cpu={currentValue.getIn(_.toPath(cpuLimitsPath))}
-              memory={currentValue.getIn(_.toPath(memoryLimitsPath))}
-              storage={currentValue.getIn(_.toPath(storageLimitsPath))}
+              cpu={currentValue?.getIn?.(_.toPath(cpuLimitsPath))}
```

Comment:
Do we need the `?` after `getIn`? I'd expect that to always work if `currentValue` isn't null.